wire-ios-data-model/Source/Model/Connection/ZMConnection+Actions.swift

```diff
@@ -24,6 +24,7 @@ public enum ConnectToUserError: Error {
     case connectionLimitReached
     case missingLegalholdConsent
     case internalInconsistency
+    case invalidDomain
```

Comment:
The tech spec was changed, we don't use this label. Have you tried to run the app and check what was deployed?